roles/nginx/tasks/configure-common.yml

```diff
@@ -33,6 +33,8 @@
 # Configure nginx Virtual Host SSL certificates
 - include: configure-ssl-cert.yml
   when: nginx_vhosts is defined
+  tags:
+    - cert:config
```

Comment:
rename `cert:config` to `nginx:config:cert`

Reply:
Done